buildpack.md

```diff
@@ -4,12 +4,13 @@ This document specifies the interface between a lifecycle program and one or mor
 
 The lifecycle program uses buildpacks to build software artifacts from source code and pack the result into an OCI image.
 
-This is accomplished in four phases:
+This is accomplished in five phases:
 
-1. **Detection,** where an optimal selection of compatible buildpacks is chosen.
-2. **Analysis,** where metadata about OCI layers generated during a previous build are made available to buildpacks.
-3. **Build,** where buildpacks use that metadata to generate only the OCI layers that need to be replaced.
-4. **Export,** where the remote layers are replaced by the generated layers.
+1. **Detection**, where an optimal selection of compatible buildpacks is chosen.
+1. **Analysis**, where metadata about OCI layers generated during a previous build are made available to buildpacks.
+1. **Generation** (optional, see the [Image Extension Interface](image-extension.md)), where image extensions generate Dockerfiles that can be used to select the runtime base image.
+1. **Build**, where buildpacks use analyzed metadata to generate only the OCI layers that need to be replaced.
```

Comment:
Isn't the Build phase also optional (in the case of extensions)?

Maybe optional is the wrong word. It's more like "restricted to extensions only" or "contingent on being a buildpack"

Reply:
The feedback from #300 was to remove this section entirely (in favor of linking back to the platform spec).